keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -57,7 +58,7 @@ class ClusterHandler(
   private val publisher: ApplicationEventPublisher,
   override val objectMapper: ObjectMapper,
   override val normalizers: List<ResourceNormalizer<*>>
-) : ResolvableResourceHandler<ClusterSpec, Set<ServerGroup>> {
+) : ResolvableResourceHandler<ClusterSpec, Map<String, ServerGroup>> {
```

Comment:
Handler now resolves `ClusterSpec` to a map by region. This diffs much better as we can tell which region is missing and get a field-by-field diff. Also see #473